lib/OrdinaryDiffEqLowOrderRK/src/algorithms.jl

```diff
@@ -276,7 +273,7 @@ function FRK65(stage_limiter!, step_limiter! = trivial_limiter!; omega = 0.0)
     FRK65(stage_limiter!, step_limiter!, False(), omega)
 end
 
-@doc explicit_rk_docstring("TBD", "RKM")
+@doc explicit_rk_docstring("To be done", "RKM")
```

Comment:
@ChrisRackauckas please write a sentence describing this method.

Reply:
Optimal Runge–Kutta Methods for First Order Pseudospectral Operators

https://www.sciencedirect.com/science/article/abs/pii/S0021999199962605

An explicit Runge-Kutta method which is designed to have good stability properties when applied to pseudospectral discretizations of hyperbolic partial differential equaitons.